lib/slop/result.rb

```diff
@@ -14,12 +14,22 @@ def initialize(parser)
       @options = parser.options
     end
 
-    # Returns an options value, nil if the option does not exist.
+    # Returns an option's value, nil if the option does not exist.
     def [](flag)
       (o = option(flag)) && o.value
     end
     alias get []
 
+    # Returns an option's value, raises KeyError if the option does not exist.
+    def fetch(flag)
+      o = option(flag)
+      if o.nil?
+        raise(KeyError.new(%Q(key not found: '#{clean_key(flag)}')))
```

Comment:
can also be expressed as

`raise KeyError, %Q(key not found: '#{clean_key(flag)}')`

Reply:
What about:

`raise KeyError, "key not found: '#{clean_key(flag)}'"`

I think that I was using `%Q` because at some point I wanted to show an error message like `key not found: "key"` but it is no longer necessary.